spec/javascripts/PaymentGateways/stripe/components/StripeCardForm.spec.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { StripeElementsForm } from 'PaymentGateways/stripe/components/StripeFormWrapper'
+import { mount } from 'enzyme'
+
+const defaultProps = {
+    stripePublishableKey: 'abcd',
+    setupIntentSecret: 'efgh',
+    billingAddressDetails: {
+        line1: '1002 Avenue de los Mexicanos',
+        line2: '',
+        city: 'South Park',
+        state: 'CO',
+        postal_code: '80440',
+        country: 'US'
+    },
+    successUrl: '/Broflovski/Residence'
```

Comment:
I know, our linter should warn about this but in the meantime...
```suggestion
  stripePublishableKey: 'abcd',
  setupIntentSecret: 'efgh',
  billingAddressDetails: {
    line1: '1002 Avenue de los Mexicanos',
    line2: '',
    city: 'South Park',
    state: 'CO',
    postal_code: '80440',
    country: 'US'
  },
  successUrl: '/Broflovski/Residence'
```